shared/templates/static/ansible/ensure_gpgcheck_never_disabled.yml

```diff
@@ -0,0 +1,19 @@
+# platform = multi_platform_rhel
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+#
+- name: Find All Yum Repositories
+  find:
+    paths: "/etc/yum.repos.d/"
+    patterns: "*.repo"
```

Comment:
I'm learning ansible as I go here.

If I understand check_mode correctly, this still should run in check mode to populate the ``yum_find`` variable.